cluster/gke/config-common.sh

```diff
@@ -28,6 +28,7 @@ GCLOUD="${GCLOUD:-gcloud}"
 CMD_GROUP="${CMD_GROUP:-beta}"
 GCLOUD_CONFIG_DIR="${GCLOUD_CONFIG_DIR:-${HOME}/.config/gcloud/kubernetes}"
 MINION_SCOPES="${MINION_SCOPES:-"compute-rw,storage-ro"}"
+MACHINE_TYPE="${MACHINE_TYPE:-ns-standard-1}"
```

Comment:
Using ns-standard-1 

`ERROR: (gcloud.container.clusters.create) ResponseError: code=400, message=machine type "ns-standard-1" does not exist.`

do you mean n1-standard1?